src/main/java/hello/controller/IssueController.java

```diff
@@ -193,40 +97,40 @@ private void fixIssue3(Integer methodId, String modifiers) {
     @GetMapping(path="/fix", produces = "application/json")
     public String fixIssue(@RequestParam(value="issueId") Integer issueId, @RequestParam(value="sandBoxUrl") String sandBoxUrl) {
         setNeo4jDriver(sandBoxUrl);
-        String USER = "neo4j";
-        String PASS = "password";
-        CodeGenPostRequest codeGenPostRequest = new CodeGenPostRequest(sandBoxUrl, USER, PASS);
+        CodeGenPostRequest codeGenPostRequest = new CodeGenPostRequest(sandBoxUrl, Constants.USER, Constants.PASS);
         Session session = driver.session();
         List<Record> result;
         switch (issueId) {
             case 1:
-                result = session.run("match (n:MethodDeclaration)<-[:member]-(:TypeDeclaration{entity_type:\"class\"})<-[:DEFINE]-(file:File) where not n.modifiers =~ \"\\\\[(public|private|protected)?(, )?(static)?(, )?(abstract)?(, )?(synchronized)?(, )?(transient|volatile)?(, )?(final)?(, )?(native)?(, )?(strictfp)?(, )?\\\\]\" return id(n) as method_id, n.line as line, n.file as file, n.col as columnName, id(file) as file_id, n.modifiers as modifiers").list();
+                result = session.run(Constants.QUERY1).list();
                 for(Record record : result) {
                     codeGenPostRequest.fileIds.add(record.get("file_id").asInt());
                     fixIssue1(record.get("method_id").asInt(), record.get("modifiers").asString());
                 }
 
-                Integer fixId = sendPostRequestCodeGen("http://codegen-cnu.ey.devfactory.com/api/codegen/", new Gson().toJson(codeGenPostRequest)).id;
+                Integer fixId = sendPostRequestCodeGen(Constants.CODEGEN_URL, new Gson().toJson(codeGenPostRequest)).id;
+
+                if (fixId == -1) {
+                    return "{message:\"some error occured\"}";
```

Comment:
Incorporate this into your response model as well. You can configure the serialiser to ignore null fields.